### DISCUSSION TITLE
自行导入的输入法方案使用词典文件

### DISCUSSION THREAD

**Discussion by ZWolken** [fcitx5-android/fcitx5-android#310] [Q&A | 问答]
想试着把自己编写的一套输入方案从 Rime 迁移到 fcitx5-android 试试，我的源文件情况（可见附图）如下：
- 码表为单个文件（仅单字+编码）
- 词语为了方便管理拆分成了多个文件（仅词汇，无对应编码）（Rime是使用了`import_tables:`来引用多个词典文件）

想问问以下两点：
1. 自行导入的输入法方案支持（同时）导入（多个）词典文件吗？
2. 如果不支持，我可以选择将词汇移动至码表的单字之后，这样子是不是必须要录入对应编码？是否有办法不用重新录入词汇的编码？

附图：
![Screenshot_2023-08-03-04-05-24-81_320a9a695de7cdce83ed5281148d6f19](https://github.com/fcitx5-android/fcitx5-android/assets/76812845/8fb319cf-0c3e-43b2-9440-4a61bacf870c)

**Comment by rocka** [comment]
1. 不支持。
2. 需要录入对应的编码。

fcitx5-chinese-addons 中的 table 方案只支持一个主码表文件，而且更多的是为类似五笔这种单纯的码表方案而优化的，并不支持词组连打/整句输入等功能。如果希望在码表中加入词语，就必须连同编码一起写到码表文件中。
